### PR TITLE
doc/dev: Fix typos in cephfs-mirroring.rst and  deduplication.rst

### PR DESCRIPTION
**Pull request by likid0**
CEPHFS: DOCS. Update Typo in Doc File cephfs-mirroring.rst
RADOS: DOCS. Update Typo in Doc File deduplication.rst

Typo Error in Doc, replace RAODS with RADOS
Signed-off-by: Daniel Parkes <dparkes@redhat.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

### CODE REVIEW

**Review by zdover23 — APPROVED** [review]

**Comment by zdover23** [comment]
@likid0: Generate a "signed-off by" line by following the instructions here: https://docs.ceph.com/en/quincy/dev/developer_guide/basic-workflow/#configuring-your-local-environment

**Review by anthonyeleven — APPROVED** [review]

**Comment by anthonyeleven** [comment]
And fill in the checklist

**Comment by zdover23** [comment]
@likid0: Now you'll need to squash the two commits into one, as described here: https://docs.ceph.com/en/quincy/start/documenting-ceph/#squash-extraneous-commits

I know that this workflow's requirements might seem obscure. Sorry about that. At least we have instructions now.

**Comment by zdover23** [comment]
Aw man.

**Comment by zdover23** [comment]
Oh, good.

**Comment by zdover23** [comment]
Don't worry about the arm64 make check and the readthedocs build, @likid0. Their failures are not necessarily going to stop this PR from being merged and backported.

**Comment by likid0** [comment]
Sorry it has been a disaster I did the first commit from the github UI the second one from my laptop with the sing-off, I just wanted to close it to see if it kicked of again the process, because the rebase looked ok

**Comment by anthonyeleven** [comment]
I have yet to successfully mix UI and CLI commits, so I feel your pain.

**Comment by likid0** [comment]
Not sure why it is failling the signed-of-by I can see it in the last rebase/squash commit right?

**Comment by zdover23** [comment]
I think that the PR's title must begin with "doc:".

**Comment by likid0** [comment]
Ok let me fix the title

**Comment by anthonyeleven** [comment]
I just aligned the PR title to the form we like to see

**Comment by likid0** [comment]
Thanks

**Comment by zdover23** [comment]
@likid0: If you write up how your expectations of this workflow lined up or didn't line up with its realities, I will try to get those thoughts into the docs.

**Comment by likid0** [comment]
Thanks, Zac, I just came in with the intention of a quick fix, and should have done the process from the laptop since the start, where I have sign-off, etc. What I can understand is why it is complaining about the current commit not passing the Signed-off-by, the current squashed commit  seems to have it in place

**Comment by anthonyeleven** [comment]
The checklist check is kinda persnickety; I think it should pass now.

**Comment by likid0** [comment]
Thanks, the email is my primary email in the account so that should be ok, let me amend the commit title

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/53519 - Reef backport

The original deduplication document was never backported into the Quincy release branch. I'll remedy that later today, and cherry-pick this commit on top of it afterwards.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/53541 - Quincy backport

**Pull request merged**
